src/helpers.js

```diff
@@ -53,6 +53,29 @@ function getDarwinApplicationVersion(bundleIdentifier) {
   return 'N/A';
 }
 
+function getAllAndroidSDKs() {
+  var androidSDKs = [];
+  try {
+    var installed = utils
+    .run(
+      '$ANDROID_HOME/tools/bin/sdkmanager --list'
```

Comment:
This doesn't actually work on my system - for whatever reason (probably because I dont use it) ANDROID_HOME isnt defined. However, I do have sdkmanager somewhere in my path, as `which sdkmanager` works, and returns `/usr/local/bin/sdkmanager`

Thoughts?

Reply:
So you could have multiple SDK managers.   Maybe we try to run the one you've identified as $ANDROID_HOME, and if that fails, we try to run the other? and if that fails, we call final failure?